sleap/gui/widgets/video.py

```diff
@@ -1732,6 +1733,211 @@ def updateEdge(self, node: QtNode):
             self.setLine(line)
 
 
+class VisibleBoundingBox(QtWidgets.QGraphicsRectItem):
+    """
+    QGraphicsRectItem for user instance bounding boxes.
```

Comment:
Put these on the same line:

```suggestion
    """QGraphicsRectItem for user instance bounding boxes.
```